scripts/st2bootstrap-el8.template.sh

```diff
@@ -11,10 +11,17 @@ DEV_BUILD=''
 USERNAME=''
 PASSWORD=''
 ST2_PKG='st2'
-ST2MISTRAL_PKG='st2mistral'
 ST2WEB_PKG='st2web'
 ST2CHATOPS_PKG='st2chatops'
 
+is_rhel() {
```

Comment:
If we use this function like `if is_rhel; then`, - it should return the status code to work properly.

Reply:
resolved in new PR: https://github.com/StackStorm/st2-packages/pull/635

Reply:
:+1: